bettercli/command.py

```
@@ -59,91 +54,140 @@ def decorator(func):
-                options[name] = op
-
-            logger.debug(f"Command.run: Calling callback with options: {options}")
-            self.callback(**options)
-
-    def add_types(self, type):
-        logger.debug(f"Command.add_types: {type=}")
-        self.types.append(type)
-
+        return [self.name] + [option.name for option in self.pos_options]
         
+    def run(self, command:'list[str]', perm_args:'dict[str, t.Any]'={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)